src/tests/unit/ingredients-dao.test.js

```diff
@@ -0,0 +1,145 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import config from 'config';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+import { getIngredientsData } from 'tests/unit/test-data';
+import { openapi } from 'utils/load-openapi';
+import { GetFilterProcessor } from 'utils/process-get-filters';
+
+const ingredientsGetParameters = openapi.paths['/ingredients'].get.parameters;
+
+sinon.restore();
+
+sinon.replace(config, 'get', () => ({ oracledb: {} }));
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.use(sinonChai);
+
+describe('test ingredients dao', () => {
+  const {
+    getIngredientsQuery,
+    emptyGetIngredientsQuery,
+    testFilters,
+    ingredientAliases,
+    testBindParams,
+    testSerializerReturn,
+    testConditionals,
+    testConnectionReturn,
+    testConnectionReturnRows,
+  } = getIngredientsData;
+  let serializeIngredientsStub;
+  let getConnectionStub;
+  let connectionSpy;
+  let ingredientsDao;
+  let getFilterProcessorStub;
+  let getFilterProcessorContructorStub;
+  beforeEach(() => {
+    connectionSpy = sinon.stub().returns(testConnectionReturn);
```

Comment:
Not a big deal, but could you name this something like `connectionStub` instead since stubs are somewhat different from spies?